crates/sui/src/sui_move/prove.rs

```diff
@@ -30,16 +32,33 @@ impl Prove {
             args.push(format!("--config={}", prover_toml.to_string_lossy()));
         }
         args.extend(opts.iter().cloned());
-        let options = move_prover::cli::Options::create_from_args(&args)?;
 
-        prove::run_move_prover(
-            build_config,
-            &rerooted_path,
-            &target_filter,
-            for_test,
-            options,
-        )?;
+        let mut options = move_prover::cli::Options::create_from_args(&args)?;
+
+        // provide Sui-specific Boogie template for the native functions to the prover.
+        options.backend.custom_natives =
```

Comment:
Should we check if the Boogie backend options are not already set from command line (from Strings) and set the ones in here as "default" only if they are not?